.github/workflows/desktop-build.yml

```
@@ -98,6 +97,10 @@ jobs:
 
           - distro: Fedora35
             package: RPM
+            test: skip
+
+          - distro: Fedora36
+            package: RPM
 
           - distro: UbuntuBionic
             package: DEB
```

Looks like you only enable tests on the most recent Linux distros and you missed this one.

```suggestion
          - distro: UbuntuBionic
            package: DEB
            test: skip
```

---

no, I specifically want our oldest build to perform tests